frontend/src/components/streamingModal/index.js

```diff
@@ -1,51 +1,31 @@
 import React from 'react'
 import { observer } from 'mobx-react'
-import { Modal, Box, Typography, Grid } from '@mui/material'
-import { alpha, useTheme } from '@mui/material/styles'
-import CloseButton from '../closeButton'
-import Button from '../button'
+import { Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
 import { useStore } from '../../store'
+import Modal from '../modal'
+import { ActionModalName } from '../../store/ActionModal'
 
 const StreamingModal = observer(() => {
   const {
-    videoPlayerStore: { showModal, onStopStreamingConfirm, onStopStreamingCancel },
-    settingsStore: { robotsSettings, currentRobotId },
+    actionModalStore: { actionModalName, showActionModal, onConfirmActionModal, onCancelActionModal },
+    settingsStore: { currentRobot },
   } = useStore()
   const theme = useTheme()
 
-  const robot = robotsSettings.find(({id}) => id === currentRobotId)
-
-  return robot && <Modal
-      open={showModal}
-      onClose={onStopStreamingCancel}
-      sx={{
-        background: alpha(theme.palette.blue[100]),
-      }}
-    >
-      <Box sx={{
-        position: 'absolute',
-        left: '50%',
-        top: '50%',
-        width: '664px',
-        transform: 'translate(-50%, -50%)',
-        backgroundColor: theme.palette.common.white,
-        padding: '44px 64px 54px 44px',
-        textAlign: 'center',
-        borderRadius: '20px',
-        boxShadow: theme.palette.boxShadow.main,
-      }}>
-        <CloseButton title='Close Guide' onClick={onStopStreamingCancel}/>
-        <Typography sx={{
-          color: theme.palette.blue[100],
-          fontSize: '24px',
-          marginBottom: '34px'
-        }}>Do you want to end the streaming robot<br/>[{robot.name}]?</Typography>
-        <Grid>
-          <Button sx={{ textTransform: 'uppercase' }} isLight onClick={onStopStreamingConfirm}>yes, I want to end streaming</Button>
-          <Button sx={{ textTransform: 'uppercase', marginLeft: '24px' }} onClick={onStopStreamingCancel}>no, I don’t </Button>
-        </Grid>
-      </Box>
-    </Modal>
+  return currentRobot && <Modal
+    showModal={actionModalName === ActionModalName.STREANMING && showActionModal}
+    onCancel={onCancelActionModal}
+    onConfirm={onConfirmActionModal}
+    confirmButtonText='yes, I want to end streaming'
```

Comment:
Shouldn't it start with a capital letter?

Reply:
button's text is capitalized by css